zp-relayer/services/commitment-watcher/router.ts

```diff
@@ -0,0 +1,70 @@
+import config from '@/configs/commitmentWatcherConfig'
+import { logger } from '@/lib/appLogger'
+import { BasePool } from '@/pool/BasePool'
+import { poolTxQueue, WorkerTx, WorkerTxType } from '@/queue/poolTxQueue'
+import { applyDenominator } from '@/utils/helpers'
+import { ValidationError } from '@/validation/api/validation'
+import cors from 'cors'
+import express, { NextFunction, Request, Response } from 'express'
+import { toBN } from 'web3-utils'
+
+export function createRouter(pool: BasePool) {
+  const router = express.Router()
+
+  router.use(cors())
+  router.use(express.urlencoded({ extended: true }))
+  router.use(express.json())
+  router.use(express.text())
+
+  router.use((err: any, req: Request, res: Response, next: NextFunction) => {
+    if (err) {
+      logger.error('Request error:', err)
+      return res.sendStatus(500)
+    }
+    next()
+  })
+
+  router.get('/address', (req, res) => {
+    res.json({ address: config.txManager.TX_ADDRESS })
+  })
+
+  router.get('/fee', (req, res) => {
+    const dInverse = toBN(1).shln(255)
+    const fee = applyDenominator(config.COMMITMENT_WATCHER_FEE, pool.denominator.xor(dInverse))
+    res.json({ fee: fee.toString(10) })
+  })
+
+  router.get('/job/:commitment', async (req, res) => {
```

Comment:
Do we need this route?

Reply:
It looks like it's not being used, but I don't see any compelling reason to remove it